Apps/Admin/Server/Services/IUserFeedbackService.cs

```diff
@@ -62,9 +63,9 @@ public interface IUserFeedbackService
         /// Associates an admin tag to a feedback.
         /// </summary>
         /// <param name="userFeedbackId">The user feedback id to be associated to the tag.</param>
-        /// <param name="tag">The admin tag.</param>
-        /// <returns>returns the associated admin tag wrapped in a request result.</returns>
-        RequestResult<UserFeedbackTagView> AssociateFeedbackTag(Guid userFeedbackId, AdminTagView tag);
+        /// <param name="adminTagIds">The admin tag ids.</param>
+        /// <returns>Returns the associated admin tag wrapped in a request result.</returns>
```

Comment:
Nit: the type of the return value is not accurately reflected in the documentation.

Reply:
Changed.